### PR TITLE
Parallelize test suite

### PR DESCRIPTION
**Pull request by Mobe91**

### CODE REVIEW

File: core/testsuite/pom.xml [hunk 1]
```diff
@@ -166,14 +169,20 @@
...
                             <systemPropertyVariables>
-                                <jdbc.url>jdbc:h2:mem:test;DB_CLOSE_DELAY=-1</jdbc.url>
+                                <jdbc.url>jdbc:h2:mem:core${surefire.forkNumber};DB_CLOSE_DELAY=-1</jdbc.url>
                                 <jdbc.user>admin</jdbc.user>
                                 <jdbc.password>admin</jdbc.password>
                                 <jdbc.driver>org.h2.Driver</jdbc.driver>
                                 <excludedGroups>com.blazebit.persistence.testsuite.base.jpa.category.NoH2,${jpa.excludedGroups}</excludedGroups>
                                 <testBasePackage>${testBasePackage}</testBasePackage>
+                                <forkCount>${surefireForkCount}</forkCount>
+                                <fork>$${surefire.forkNumber}</fork>
```
Comment (beikov) [inline core/testsuite/pom.xml:184]: Double-dollar?

File: core/testsuite/src/test/java/com/blazebit/persistence/testsuite/DateTruncTest.java [hunk 2]
```diff
@@ -46,18 +47,21 @@
 
     public DateTruncTest() {
         c1 = Calendar.getInstance();
+        c1.setTimeZone(TimeZone.getTimeZone("UTC"));
```
Comment (beikov) [inline core/testsuite/src/test/java/com/blazebit/persistence/testsuite/DateTruncTest.java:50]: Why are you setting the timezone here and below?

File: core/testsuite/src/test/java/com/blazebit/persistence/testsuite/DateAddTest.java [hunk 3]
```diff
@@ -46,18 +46,21 @@
 
     public DateAddTest() {
         c1 = Calendar.getInstance();
+        c1.setTimeZone(TimeZone.getTimeZone("UTC"));
```
Comment (beikov) [inline core/testsuite/src/test/java/com/blazebit/persistence/testsuite/DateAddTest.java:49]: Why are you setting the timezone here and below?

**Review by beikov — CHANGES_REQUESTED** [review]

File: core/testsuite/pom.xml (hunk 1 above)
Comment (Mobe91) [reply core/testsuite/pom.xml:184]: Required

**Review by beikov — APPROVED** [review]

**Pull request merged**